src/tools/rsync.cc

```diff
@@ -926,8 +926,12 @@ QList<QByteArray> Rsync::calculateRsyncRulesFromIncludeRules( const BackupSelect
         filters.clear();
     }
 
-    if (filters.contains("+"))
-      filters << convertRuleToByteArray( "**",false );
+    foreach (QByteArray filter, filters) {
+        if (filter.contains("+")) {
```

Comment:
wouldn't it be safer to use `filter.trimmed().startsWith('+')` instead or are "+" in filenames somehow encoded?